lmdeploy/model.py

```diff
@@ -56,6 +56,7 @@ def chat_template(self):
         return model
 
 
+@MODELS.register_module(name='mistral')
```

Comment:
What's the chat template?

Reply:
I don't know. Mistral does not provide a chat model.

Reply:
Mistral has models like [mistralai/Mistral-7B-Instruct-v0.1](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.1),  [mistralai/Mistral-7B-Instruct-v0.2](https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.2).
I write the [chat template](https://github.com/InternLM/lmdeploy/blob/25e905ffd41e6c26253aceff620c16e228acc9db/lmdeploy/model.py#L943) in PR #1133. Could you check and review the PR?

Reply:
can the chat template in #1133 be merged into this one?